app.js

```diff
@@ -0,0 +1,9 @@
+import Fastify from 'fastify'
+
+export const app = Fastify({
+  logger: true
```

Comment:
It should have an option to silent log - for example in test run

Reply:
hopefully done ;)